lib/flux/settings-reducer.js

```diff
@@ -49,11 +49,25 @@ const markdownEnabled = ( state = false, action ) => {
 	return action.markdownEnabled;
 };
 
+const sourceList = ( state = { width: 330, isResizing: false, canResize: false }, { type, width, canResize, isResizing } ) => {
```

Comment:
even better yet, why store complex state in a single reducer when we have `combineReducers()`?

```js
const width = ( state = 330, { type, width } ) =>
	type === SOURCE_LIST_WIDTH_SET && state !== width
		? width
		: state;

const isResizing = ( state = false, { type, isResizing } ) =>
	type === SOURCE_LIST_IS_RESIZING_SET && state !== isResizing
		? isResizing
		: state;

const canResize = ( state = false, { type, canResize } ) =>
	type === SOURCE_LIST_CAN_RESIZE_SET && state !== canResize
		? canResize
		: state;

export default combineReducers( {
	canResize,
	isResizing,
	width,
} );
```

This seems much more testable and Redux-y